### ISSUE TITLE
MOTR-270 Tool for local validation of experimentalDetails files

### ISSUE THREAD

**Issue by cteng585** [MoTrPAC/motrpac-frontend#78]
Short script to check that experimentalDetails files that are being submitted by metabolomics sites are compliant with this template LOCALLY:  https://docs.google.com/document/d/1zHURGJgTK94UfT5ICjsZo7wLdKR3KTt3-I70xrk0ORY/edit?usp=sharing

It will be made clear to sites that this tool will NOT be maintained after PASS1A and PASS1B data is brought into compliance with the template.

**Comment by jimmyzhen** [comment]
@cteng585, is this GitHub issue meant to be for this repo?

**Comment by cteng585** [comment]
No, this was a mistake from when I was originally trying out Zenhub
